R/tm_a_pca.R

```diff
@@ -706,8 +710,8 @@ srv_a_pca <- function(id, data, reporter, filter_panel_api, dat, plot_height, pl
         dev_labs <- list(color = varname_w_label(resp_col, ANL))
 
         scales_biplot <- if (is.character(response) ||
```

Comment:
It looks like superlinter or some cleaning mechanism removed whitespaces for you. Now this `if` statement looks a bit odd. What if we move `is.character(response) ||` to the newline so it looks more readable? Can you apply this comment also to `geom_bar` later below, where you can move `aes_string` to the new line?

Reply:
I moved it to new line but it affected line below `is.character(response) ||` 
https://github.com/insightsengineering/teal.modules.general/pull/534/commits/3f98314485f1a0e5959bda69823c7a50d2b458fb

same with switch statement as well it reverts back changes.